Extension/c_cpp_properties.schema.json

```diff
@@ -14,7 +14,11 @@
                         "description": "Platform name. Mac, Linux, or Win32 are the defaults unless a custom platform is added.",
                         "type": "string"
                     },
-                    "compileCommands":{
+                    "compilerPath": {
```

Comment:
need to add code in client.ts to remove this setting from telemetry.

Reply:
Ah, yeah.

Reply:
Wait, we don't send telemetry for this.

Reply:
You're right.  the sliver of code I saw looked a lot like package.json, so I was in the wrong frame of mind.  Please ignore the comment.